test/parallel/test-dgram-empty-packet.js

```diff
@@ -28,7 +29,6 @@ client.bind(0, function() {
     callback();
   });
 
-  const port = this.address().port;
```

Comment:
IMHO this is best left here since it's declared right above the only place that it's used.